src/projects/EnsureThat/CollectionArg.cs

```diff
@@ -382,5 +382,50 @@ public T[] HasAny<T>([ValidatedNotNull]T[] value, [NotNull] Func<T, bool> predic
 
             return value;
         }
+
+        [NotNull]
+        [DebuggerStepThrough]
+        public IList<T> Contains<T>([NotNull, ValidatedNotNull]IList<T> value, T item, [InvokerParameterName] string paramName = Param.DefaultName)
+        {
+            if (!Ensure.IsActive)
+                return value;
+
+            Ensure.Any.IsNotNull(value, paramName);
+
+            if (!value.Contains(item))
+                throw new ArgumentException(ExceptionMessages.Collections_Contains_Failed, paramName);
+
+            return value;
+        }
+
+        [NotNull]
+        [DebuggerStepThrough]
+        public ICollection<T> Contains<T>([NotNull, ValidatedNotNull]ICollection<T> value, T item, [InvokerParameterName] string paramName = Param.DefaultName)
+        {
+            if (!Ensure.IsActive)
+                return value;
+
+            Ensure.Any.IsNotNull(value, paramName);
+
+            if (!value.Contains(item))
+                throw new ArgumentException(ExceptionMessages.Collections_Contains_Failed, paramName);
+
+            return value;
+        }
+
+        [NotNull]
+        [DebuggerStepThrough]
+        public T[] Contains<T>([NotNull, ValidatedNotNull]T[] value, T item, [InvokerParameterName] string paramName = Param.DefaultName)
+        {
+            if (!Ensure.IsActive)
+                return value;
+
+            Ensure.Any.IsNotNull(value, paramName);
+
+            if (!value.Contains(item))
```

Comment:
Half the point of exposing the [] one is the perf benefits of [], which are lost by using this linq method :( LinqIsNotQuick